bob/src/api/http.rs

```diff
@@ -523,3 +562,37 @@ impl From<Status> for StatusExt {
         }
     }
 }
+
+impl From<std::io::Error> for StatusExt {
+    fn from(err: std::io::Error) -> Self {
+        Self {
+            status: match err.kind() {
+                ErrorKind::NotFound => Status::NotFound,
+                _ => Status::BadRequest,
+            }, // TODO: Complete match
+            ok: false,
+            msg: err.to_string(),
+        }
+    }
+}
+
+impl From<bob_common::error::Error> for StatusExt {
```

Comment:
I think it's better to do this with `match`. Maybe you need to change visibility scope of bob Error/ErrorKind.